l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -158,6 +158,8 @@ def _setFormatoTrasmissione(self, cr, uid, company, context=None):
     def _setCodiceDestinatario(self, cr, uid, partner, context=None):
         if context is None:
             context = {}
+        #TODO check iPa code is required only for P.A. company
+        # but xml invoice can be make also for private company
```

Comment:
@eLBati  yes , but only if your xml is made for an P.A. and the communication is under SDI ,but if you want send an xml invoice to private company IPA code is not required. For Example big private company like Enel only accept xml invoices http://www.acquisti.enel.it/en-GB/doc/Comunicazione_Fornitori_webedi.pdf